core/src/main/java/org/apache/accumulo/core/util/threads/ThreadPoolNames.java

```diff
@@ -0,0 +1,54 @@
+package org.apache.accumulo.core.util.threads;
+
+public enum ThreadPoolNames {
+
+  ACCUMULO_POOL_PREFIX("accumulo.pool"),
+  BATCH_SCANNER_POOL_NAME("accumulo.pool.client.batch.reader.scanner"),
```

Comment:
Just a suggestion, one that can be ignored. It might make sense to remove `_NAME` from the enum name, as the proper name currently is `ThreadPoolNames.BATCH_SCANNER_POOL_NAME`. It might be redundant.

Reply:
I went ahead with this refactor since it was pretty trivial